package.json

```diff
@@ -17,9 +17,15 @@
     "build-storybook": "build-storybook"
   },
   "dependencies": {
+    "@algolia/autocomplete-core": "^1.17.2",
+    "@algolia/autocomplete-js": "^1.17.2",
+    "@algolia/autocomplete-plugin-query-suggestions": "^1.17.2",
+    "@algolia/autocomplete-plugin-recent-searches": "^1.17.4",
+    "@algolia/autocomplete-theme-classic": "^1.17.2",
     "@mdx-js/loader": "^3.0.1",
     "@next/mdx": "12.3.1",
     "@tanstack/react-query": "^5.17.19",
+    "algoliasearch": "^4.24.0",
```

Comment:
what is this for?

Reply:
This is a package for 3rd party integration algolia